src/engine/renderer/gl_shader.cpp

```diff
@@ -390,6 +390,55 @@ static std::string BuildDeformSteps( deformStage_t *deforms, int numDeforms )
 	return steps;
 }
 
+struct addedExtension_t {
+	bool &available;
+	int minGlslVersion;
+	std::string name;
+};
+
+// Basic version declaration.
```

Comment:
"Basic" is not totally self-explanatory so it would be good to say that this for vertex + fragment shaders.

Reply:
I used `FragmentVertex` instead.